.github/workflows/unit-tests.yaml

```diff
@@ -22,4 +22,4 @@ jobs:
 
       - name: Run unit tests
         run: |
-          python -m unittest process_report/tests/unit_tests.py
+          python -m unittest
```

Comment:
nit (can be fixed in follow-up): I would still keep this pointing to `process_report/tests/unit` rather than the entire root directory.